### PR TITLE
Update sdk

### PR DESCRIPTION
**Pull request by ozgunozerk**
After this, I'm going to issue a release

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ tracing-error = "0.2.0"
 tracing-subscriber = { version = "0.3.16", features = ["env-filter"] }
 whoami = "1"
 
-subspace-sdk = { git = "https://github.com/subspace/subspace-sdk", rev = "ae2c1f6bd975f3471ee0d7243f2753b10593e194" }
+subspace-sdk = { git = "https://github.com/subspace/subspace-sdk", rev = "dfa0c919368bc0beab9dff5c552a771318eddd92" }
```
Comment (i1i1) [inline Cargo.toml:38]: It is best to stick to commits in `gemini-3d-maintenance` branch for now, as CLI will stop working otherwise

**Review by i1i1 — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (ozgunozerk) [reply Cargo.toml]: ah yeah, thanks for the catch

**Comment by i1i1** [comment]
Can you squash commits?

**Comment by ozgunozerk** [comment]
I'm always squash merging these kind of PR's

**Review by i1i1 — APPROVED** [review]

**Pull request merged**